documentation/operators/src/testing/gateway-probe.md

```diff
@@ -0,0 +1,79 @@
+# Nym Gateway Probe
+
+Nym Node operators running Gateway functionality are already familiar with the monitoring tool [Harbourmaster.nymtech.net](https://harbourmaster.nymtech.net). Under the hood of Nym Harbourmaster runs iterations of `nym-gateway-probe` doing various checks and displaying the results on the interface. Operators don't have operators rely on the probe run by Nym and wait for the data refresh. With `nym-gateway-probe` everyone can check any Gateway on the network. In one command the client queries data from:
```

Comment:
" Operators don't have operators rely on the probe run by Nym and wait for the data refresh." 

What do you mean here?

Reply:
typo, fixing